core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
> It does sound like you have some specific rules/guidelines/axioms you'd like the project to adopt, would you consider opening up an issue for them?

I could. But, since several of the things we'd propose have already been decided against by the core team in the few PRs we've seen I think it may not be the best use of the Solidus team's time (or ours, of course).

To be specific here, probably our most fundamental axiom in projects we run…

>  Use the least powerful primitive.

…has already been dismissed a couple times. 

I'm happy to discuss this further in Slack so we don't keep polluting this thread. Just seemed important to answer your question here first.